src/status_im/config.cljs

```diff
@@ -134,3 +134,10 @@
 ;; Alert banners are disabled for debug builds because alert banners overlay
 ;; interfere with react-native debug tools, such as inspector and Perf monitor
 (def enable-alert-banner? (enabled? (get-config :ENABLE_ALERT_BANNER "0")))
+
+(goog-define STATUS_BACKEND_ENABLED "0")
+;; e.g. 127.0.0.1:58935
+(goog-define STATUS_BACKEND_SERVER_HOST "")
```

Comment:
It's not a big deal @qfrank, just a bit more flexible. Having two variables would make the comment unnecessary. A lot of things are configured with host and port separately, but not everything of course.

For instance:

```sh
# Somewhere far away
export STS_BACKEND_PORT=60000

# status-go
PORT=$STS_BACKEND_PORT make run-status-backend

# status-mobile
STATUS_BACKEND_SERVER_PORT=$STS_BACKEND_PORT make run-clojure
```

This would be nice because I can configure the port once somewhere and reuse and I can easily change the port from a single place.

But again, not too important, just a nice to have.